validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 7           | Label worker nodes with `node.longhorn.io/create-default-disk=true` | | Nodes are properly labeled                  |
+| 8           | Complete installation with custom configuration | | Installation succeeds with custom settings   |
+| 9           | Verify custom settings in Longhorn UI | | Settings reflect custom configuration        |
+| 10          | Create test volume and verify replica count | | Volume created with 2 replicas               |
+| 11          | Check custom data path utilization on nodes | | `/var/lib/longhorn-custom` directory is used |
+
+### Longhorn UI Access Through Rancher
+
+Verify seamless access to Longhorn UI through Rancher interface with proper authentication and session management.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Navigate to cluster view in Rancher dashboard | | Cluster overview displays |
+| 2 | Verify "Longhorn" appears in left navigation menu | | Longhorn menu item is visible |
+| 3 | Click "Longhorn" menu item | | Longhorn resources display in Rancher |
+| 4 | Click the "Longhorn" button in the Overview section | | Longhorn UI opens in new tab/window |
+| 5 | Navigate to "Volume" section in Longhorn UI | | Volume management page loads |
+| 6 | Navigate to "Node" section in Longhorn UI | | Node management page loads |
+| 7 | Navigate to "Setting" section in Longhorn UI | | Settings page loads |
+| 8 | Return to Rancher tab and verify session maintained | | Rancher session still active |
+| 9 | Create test volume from Longhorn UI | | Volume creation succeeds |
+| 10 | Return to Rancher and verify volume appears | | Volume visible in Rancher Longhorn section |
+
+### RBAC Integration Testing
+
+Test Role-Based Access Control integration between Rancher and Longhorn for different user permission levels.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Create cluster admin user in Rancher | | User created with cluster-admin role |
+| 2 | Create project member user in Rancher |  | User created with project-member role |
+| 3 | Create read-only user in Rancher | | User created with read-only permissions |
+| 4 | Login as cluster admin and access Longhorn UI | | Full access to all Longhorn features |
+| 5 | Test volume creation and deletion as cluster admin | | Operations succeed without restrictions |
+| 6 | Login as project member and access Longhorn UI | | Access granted with project scope |
+| 7 | Test volume operations within project scope | | Operations succeed within assigned project |
+| 8 | Login as read-only user and access Longhorn UI | | Read-only access granted |
+| 9 | Attempt volume creation as read-only user | | Operation blocked with appropriate error message |
+| 10 | Verify RBAC consistency across both UIs | | Permissions enforced consistently |
+
+### Volume Creation Through Rancher Workloads
```

Comment:
would reword to something like `Create and mount PVC` or more specifically `Create and Mount ReadWriteOnce PVC`

Reply:
Why? Does the title not make it clear?

Reply:
yeah I had to read the steps to know what is going on. Since the goal would be to test the storage side, including 'workloads' in the title made it somewhat confusing imo.